app/views/councillor_contributions/new.html.haml

```diff
@@ -5,5 +5,9 @@
       - @councillor_contribution.suggested_councillors.each do |s|
         = render 'contribution_form_input', f: f, suggested_councillor: s
       %button{formaction: new_authority_councillor_contribution_path(@authority.short_name_encoded), class: "button"} Add another councillor
+    .councillor-contribtution-source
+      %p Where did you find this information? please include a link if possible
+      = f.label :name, "Source URL"
+      = f.text_field :source_url
```

Comment:
@hisayohorie Do you recon we should make a bigger text area here to encourage people to add more information if they want? 

Following on from what @henare wrote https://github.com/openaustralia/planningalerts/issues/1221#issuecomment-323242217 I recon that would encourage people a bit more.

> This also allows for input we're not expecting, i.e. not a URL, that might be something like, "I found the information in my local newspaper".

> I believe this permissive design approach is generally a good idea but especially with ideas like this that are at their early stages

If we do want to make a more spacious input for people, like our comment form, we could use `text_area` rather than `text_field` here.

![screen shot 2017-08-22 at 12 43 47 pm](https://user-images.githubusercontent.com/1239550/29546406-9894d88e-8737-11e7-96ae-17a495c1af22.png)